### PR TITLE
feat: embedding model in env and seeding chat completion

### PR DESCRIPTION
**Pull request by Jaszkowic**

### CODE REVIEW

File: .envrc.sample
Comment (ff6347) [inline .envrc.sample:1]: Why remove the example values from the sample and why remove the LOG_LEVEL and dev env DANGEROUSLY_ALLOW_CORS_FOR_ALL_ORIGINS? They are both still in use.

I tend to see these sample files as documentation.

File: README.md
Comment (ff6347) [inline README.md:1]: same here? Why remove the example values

**Review by ff6347 — CHANGES_REQUESTED** [review]

File: README.md
Comment (Jaszkowic) [reply README.md:1]: sorry, that was by accident, fixed

File: .envrc.sample
Comment (Jaszkowic) [reply .envrc.sample:1]: sorry, that was by accident, fixed

Comment (ff6347) [inline .envrc.sample:1]: I really would like to also have the sample values 🥺

Comment (Jaszkowic) [reply .envrc.sample:1]: your wish has been granted

**Review by ff6347 — APPROVED** [review]

**Pull request merged**